gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/NativeDistributions.kt

```diff
@@ -27,6 +27,7 @@ open class NativeDistributions @Inject constructor(
     var vendor: String? = null
     var packageVersion: String? = null
     val appResourcesRootDir: DirectoryProperty = objects.directoryProperty()
+    val packagingResourcesRootDir: DirectoryProperty = objects.directoryProperty()
```

Comment:
I'm now adding `packagingResourcesRootDir` to the DSL similar to how `appResourcesRootDir` is implemented.